src/frontend/src/lib/components/ui/IdentitySwitcher.svelte

```diff
@@ -17,106 +16,168 @@
-              {identity.name ?? identity.identityNumber}
-            </div>
-            <div class="text-text-tertiary font-normal" aria-hidden="true">
-              {#if nonNullish(date)}
-                <span>{$t`${name} | Created ${date}`}</span>
+    </button>
+  </div>
+  <div class="mb-4 flex flex-col gap-1.5">
+    <ul class="contents">
+      {#each identities as identity}
+        {@const name = lastUsedIdentityTypeName(identity)}
+        {@const date = nonNullish(identity.createdAtMillis)
+          ? $formatDate(new Date(identity.createdAtMillis), {
+              dateStyle: "short",
+            })
+          : undefined}
+        <li class="contents">
+          <ButtonCard
+            onclick={() => handleSwitchIdentity(identity.identityNumber)}
+            class={[
+              selected === identity.identityNumber &&
+                "!ring-border-brand disabled:!ring-border-disabled !ring-2",
+            ]}
+          >
+            <Avatar size="sm">
+              {#if switchingToIdentity === identity.identityNumber}
+                <ProgressRing />
               {:else}
-                <span>{name}</span>
+                <UserIcon class="size-5" />
               {/if}
+            </Avatar>
+            <div class="flex flex-col text-left text-sm">
+              <div class="text-text-primary font-semibold">
+                {identity.name ?? identity.identityNumber}
+              </div>
+              <div class="text-text-tertiary font-normal" aria-hidden="true">
+                {#if nonNullish(date)}
+                  <span>{$t`${name} | Created ${date}`}</span>
+                {:else}
+                  <span>{name}</span>
+                {/if}
+              </div>
             </div>
-          </div>
-          {#if selected === identity.identityNumber}
-            <Checkbox
-              checked
-              size="md"
-              class="pointer-events-none mr-1 ml-auto !rounded-full"
-              tabindex={-1}
-              aria-hidden
-            />
-          {/if}
-        </ButtonCard>
-      </li>
-    {/each}
-  </ul>
-  <ButtonCard onclick={useAnotherIdentity}>
-    <FeaturedIcon size="sm">
-      <!-- Design also uses smaller icon -->
-      <!-- It looks better than the same size as the others -->
-      <UserPlusIcon class="size-4" />
-    </FeaturedIcon>
-    <span>{$t`Use another identity`}</span>
-  </ButtonCard>
-</div>
-{#if onLogout}
-  <Button onclick={onLogout} variant="tertiary">
-    <LogOutIcon class="size-5" />
-    <span>{$t`Sign Out`}</span>
-  </Button>
-{:else if nonNullish(manageIdentityUrl)}
-  <Button href="/manage" target="_blank" variant="tertiary">
-    <Settings class="size-5" />
-    <span>{$t`Manage Identity`}</span>
-  </Button>
-{/if}
+            {#if selected === identity.identityNumber}
+              <Checkbox
+                checked
+                size="md"
+                class="pointer-events-none mr-1 ml-auto !rounded-full"
```

Comment:
`pointer-events-none` is another class responsible for more than just visualization, right?